src/FlapperUniV2.sol

```diff
@@ -103,8 +102,7 @@ contract FlapperUniV2 {
         wards[msg.sender] = 1;
         emit Rely(msg.sender);
 
-        // Initial values for safety
-        hop  = 1 hours;
+        // Initial value for safety
```

Comment:
neat - can put the comment in the same line now.

Reply:
Changed in 79a0021c